ND_Supporting_Document_3_0.adoc

```diff
@@ -1852,8 +1726,8 @@ Remark: Some systems might require the presence of the SAN extension. In this ca
 . The evaluator shall demonstrate that using an invalid certificate results in the function failing as follows:
 [loweralpha, start=1]
 .. Test 1: Using the administrative guidance, the evaluator shall load a CA certificate or certificates needed to validate the presented certificate used to authenticate an external entity and demonstrate that the function succeeds and a trusted channel can be established.
-.. Test 2: The evaluator shall then change the presented certificate(s) so that validation fails and show that the certificate is not automatically accepted. The evaluator shall repeat this test to cover the selected types of failure defined in the SFR (i.e. the selected ones from failed matching of the reference identifier, failed validation of the certificate path, failed validation of the expiration date, failed determination of the revocation status). The evaluator shall perform the action indicated in the SFR selection observing the TSF resulting in the expected state for the trusted channel (e.g. trusted channel was established) covering the types of failure for which an override mechanism is defined.
-.. Test 3 [conditional]: The purpose of this test to verify that only selected certificate validation failures could be administratively overridden. If any override mechanism is defined for failed certificate validation, the evaluator shall configure a new presented certificate that does not contain a valid entry in one of the mandatory fields or parameters (e.g. inappropriate value in extendedKeyUsage field) but is otherwise valid and signed by a trusted CA. The evaluator shall confirm that the certificate validation fails (i.e. certificate is rejected), and there is no administrative override available to accept such certificate.
+.. Test 2 [conditional]: If administrator override is selected, the evaluator shall then change the presented certificate(s) so that validation fails due to a failure to to determine revocation status and show that the certificate is not automatically accepted. The evaluator shall perform the action indicated in the SFR selection observing the TSF resulting in the expected state for the trusted channel (e.g. trusted channel was established).
```

Comment:
Updated FCS_DTLSC_EXT.1.3 and FCD_TLSC_EXT.1.3 Tests for clarify, largely following the recommendations.

1. I'm trying to keep edits minimal right now. I don't think this has been a big problem, but agree it would be good to clarify in the future.
2. I think there are too many nuances around revocation for the test to go into details. IMO, it's the evaluator's responsibility to determine what "inability to determine revocation status" means for a specific TOE.